config.js

```diff
@@ -49,6 +49,7 @@ exports.updateFromArgv = function() {
         .string('greeting')
         .describe('farewell', 'Thank you message to appear in chat at end of game (ex: -Thank you for playing-)')
         .string('farewell')
+        .describe('farewell_score', 'Send the score according to the bot at the end of the game')
```

Comment:
so far no gtp2ogs option uses an underscore `_`, we may want to keep it as that, but @Dorus and @roy7 most likely know better than me

Reply:
also, if the purpose is just to send the score, your bot should already be set up to sent that information at the last move of `config.aichat`

if the purpose is to send a specific bot chat at the end of the game, you may want to use a more general `farewellaichat` option to send any text, not just score
in game.js you'd just have to send this text before killing bot

Reply:
The bot doesn't know it's the last move, since it gets murdered on a gameOver (resign) event.
and quit is ambiguous as well.